pages/Ethereum/Identity.md

```diff
@@ -1,48 +1,28 @@
 ---
-name: Overview
-route: /Identity/Overview
-menu: Identity
+name: Identity
+route: /Ethereum/Identity
+menu: "DSNP: Ethereum"
 ---
 
 # Identity
 
-This specification is intended to cover the concept of identity within the protocol and how we represent it.
-
 ## Purpose
 
 1. Provide the interface for a DSNP identity.
 1. Specify the delegation model and related interface.
 1. Specify the ownership model and related interface.
 1. Provide the list of EIPs that must be supported by a DSNP compatible identity.
 
-## Assumptions
-
-* Identity is on Ethereum for Betanet.
-* Identity will need to be moved from Betanet to Mainnet.
-
 ## Details
 
-Identity is comprised of several parts:
-
-* Ownership
-* Delegation
-* Identification
-* Discovery
-
-Additional constraints were placed upon any solution:
-
-* Pseudo Anonymity: A consistent identifier that defaults to disconnected from real world identity
-* Flexibility: An interface that can produce multiple solutions
-* Extensibility: A default implementation that can be updated over time as DSNP changes
-
 ## Ownership
 
-On the blockchain ownership is determined by the ability to sign messages.
-To this end, ownership of an identity is also determined by the address of the creator.
-(Options for allowing a delegate to create an identity on behalf of someone is described in the [Identity Factory](/Identity/Factory).)
+Ownership of an identity on Ethereum is determined by the address of the signer of the content.
 
 Creation of any new identity MUST be authorized by the owner's address.
-The official [Identity Factory](/Identity/Factory) must be used for initial creation of a new identity.
+The official [Identity Factory](/Ethereum/IdentityFactory) must be used for initial creation of a new identity.
```

Comment:
Just curious: how would any observer of the chain be able to tell the difference between an identity that was created with the official Identity Factory and one that was compatibly created using some other means?  (Or maybe I'm misunderstanding what "Factory" means here?

Reply:
Ah here is that requirement. This is a mistake. It is not required.

```suggestion
The official [Identity Factory](/Ethereum/IdentityFactory) is provided for initial creation of a new identity.
```